### PR TITLE
Add tabs for Student List and Session List

### PR DESCRIPTION
**Pull request by fatin99**

### CODE REVIEW

File: src/main/resources/view/MainWindow.fxml [hunk 1]
```diff
@@ -33,27 +35,33 @@
...
         </StackPane>
 
-        <VBox fx:id="studentList" styleClass="pane-with-border" minWidth="340" prefWidth="340" VBox.vgrow="ALWAYS">
-          <padding>
-            <Insets top="10" right="10" bottom="10" left="10" />
-          </padding>
-          <StackPane fx:id="studentListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
-        </VBox>
-
         <StackPane fx:id="statusbarPlaceholder" VBox.vgrow="NEVER" />
```
Comment (PotatoCombat) [inline src/main/resources/view/MainWindow.fxml:70]: Maybe the status bar could be below the tab pane?

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -33,27 +35,33 @@
...
+            <TabPane prefHeight="433.0" prefWidth="200.0" tabClosingPolicy="UNAVAILABLE">
```
Comment (PotatoCombat) [inline src/main/resources/view/MainWindow.fxml]: Add VBox.vgrow="ALWAYS" in here to remove the white screen bug

File: src/main/java/seedu/address/model/session/SessionType.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package seedu.address.model.session;
+
+/**
+ * Represents a session type. Session types follows the same specifications as the TSS.
+ * Example session types include: Tutorial, Grading, Consultation, etc.
+ */
+public enum SessionType {
```
Comment (PotatoCombat) [inline src/main/java/seedu/address/model/session/SessionType.java]: Good thinking making this into an enum class! We might need to do the same for the other enums

File: src/main/java/seedu/address/model/session/Session.java [hunk 4]
```diff
@@ -10,20 +10,6 @@
...
-     */
-    public enum SessionType {
-        SESSION_TYPE_TUTORIAL,
-        SESSION_TYPE_LAB,
-        SESSION_TYPE_CONSULTATION,
-        SESSION_TYPE_GRADING,
-        SESSION_TYPE_PREPARATION,
-        SESSION_TYPE_OTHER
-    }
-
```
Comment (PotatoCombat) [inline src/main/java/seedu/address/model/session/Session.java]: Should probably let @Eclmist know that this has been extracted out!

File: src/main/resources/view/MainWindow.fxml [hunk 5]
```diff
@@ -6,13 +6,15 @@
...
+<?import javafx.scene.control.TabPane?>
 <?import javafx.scene.image.Image?>
+<?import javafx.scene.layout.AnchorPane?>
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
+<?import javafx.stage.Stage?>
 
-<fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
-         title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+<fx:root minHeight="600" minWidth="450" onCloseRequest="#handleExit" title="Address App" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (PotatoCombat) [inline src/main/resources/view/MainWindow.fxml]: For now can change back the java fx version? Will need to re-check if whether they've updated this in AB-3

File: src/main/resources/view/MainWindow.fxml [hunk 6]
```diff
@@ -33,27 +35,33 @@
...
+                <Tab fx:id="studentListTab" text="Student List">
+                  <content>
+                    <StackPane fx:id="studentListPanelPlaceholder" VBox.vgrow="ALWAYS" />
+                  </content>
+                </Tab>
+                <Tab fx:id="sessionListTab" text="Session List">
+                  <content>
+                    <StackPane fx:id="sessionListPanelPlaceholder" VBox.vgrow="ALWAYS" />
+                  </content>
+                </Tab>
```
Comment (PotatoCombat) [inline src/main/resources/view/MainWindow.fxml]: Thanks for making the tabs!

File: src/main/resources/view/MainWindow.fxml (hunk 2 above)
Comment (fatin99) [reply src/main/resources/view/MainWindow.fxml]: the status bar is not part of the whitescreen bug, but anyway its already resolved

File: src/main/resources/view/MainWindow.fxml (hunk 5 above)
Comment (fatin99) [reply src/main/resources/view/MainWindow.fxml]: Oh, it auto changed on scene builder. My bad

File: src/main/resources/view/MainWindow.fxml [hunk 7]
```diff
@@ -62,7 +63,8 @@
           </tabs>
         </TabPane>
 
-        <StackPane fx:id="statusbarPlaceholder" VBox.vgrow="ALWAYS" />
+        <StackPane fx:id="statusbarPlaceholder" VBox.vgrow="NEVER" />
```
Comment (PotatoCombat) [inline src/main/resources/view/MainWindow.fxml:70]: Hmm this change makes resizing the window not work

File: src/main/resources/view/MainWindow.fxml [hunk 8]
```diff
@@ -14,7 +14,8 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.stage.Stage?>
 
-<fx:root minHeight="600" minWidth="450" onCloseRequest="#handleExit" title="Address App" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
+         title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
```
Comment (PotatoCombat) [inline src/main/resources/view/MainWindow.fxml]: Maybe can change this commit to be just to restore the old java fx version?

File: src/main/resources/view/MainWindow.fxml (hunk 2 above)
Comment (PotatoCombat) [reply src/main/resources/view/MainWindow.fxml]: Eh I show u what I mean

**Review by chuayijing — APPROVED** [review]
looks good for merging !

File: src/main/java/seedu/address/logic/parser/SessionCommandParser.java [hunk 9]
```diff
@@ -44,13 +44,15 @@ public Command parseCommand(String userInput) throws ParseException {
...
             return new AddSessionCommandParser().parse(arguments);
+            //throw new ParseException(String.format(UNIMPLEMENTED_CODE_FORMAT, "Add session commands"));
 
         case CommandWords.DELETE_MODEL:
             // return new DeleteSessionCommandParser().parse(arguments);
             throw new ParseException(String.format(UNIMPLEMENTED_CODE_FORMAT, "Delete session commands"));
 
         case CommandWords.EDIT_MODEL:
             return new EditSessionCommandParser().parse(arguments);
+            // throw new ParseException(String.format(UNIMPLEMENTED_CODE_FORMAT, "Edit session commands"));
```
Comment (PotatoCombat) [inline src/main/java/seedu/address/logic/parser/SessionCommandParser.java]: Actually can remove already. But its no issue I'll remove this in my next pr.

**Review by PotatoCombat — APPROVED** [review]

**Comment by PotatoCombat** [comment]
LGTM!

**Pull request merged**